CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
   * `--experimental.distributor.user-subring-size`
 * [FEATURE] Added flag `-experimental.ruler.enable-api` to enable the ruler api which implements the Prometheus API `/api/v1/rules` and `/api/v1/alerts` endpoints under the configured `-http.prefix`. #1999
 * [FEATURE] Added sharding support to compactor when using the experimental TSDB blocks storage. #2113
+* [FEATURE] Add ability to override configuration settings using environment variables. #2147
```

Comment:
I would replace `configuration settings` with `YAML config file settings` to be more clear.